### PR TITLE
Remove "-of" from the input path name pattern

### PR DESCRIPTION
**Pull request by monocongo**
Remove "-of" from the regex pattern for the input path at line 207 since the output files from the code in lines 178-192 doesn't actually produce files with "of-" in their names (as listed in lines 196-201).

### CODE REVIEW

File: research/object_detection/g3doc/using_your_own_dataset.md [hunk 1]
```diff
@@ -204,6 +204,6 @@ which can then be used in the config file as below.
 
 ```bash
 tf_record_input_reader {
-  input_path: "/path/to/train_dataset.record-?????-of-00010"
```
Comment (pkulzc) [inline research/object_detection/g3doc/using_your_own_dataset.md]: I think this is actually needed. Instead you should update the file list above.

Comment (monocongo) [reply research/object_detection/g3doc/using_your_own_dataset.md]: You are correct, thanks for your sharp attention and catching my mistake.

**Comment by monocongo** [comment]
I have updated the file accordingly as requested by @pkulzc, the documentation should now be accurate.

**Comment by jaeyounkim** [comment]
@pkulzc Please review this pull request again.

**Pull request closed without merging**